slack_bot/plugins/issue.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import requests
+
+
+ISSUE_API = "https://api.github.com/repos/{org}/{repo}/issues"
+REPO_API = "https://api.github.com/orgs/{org}/repos"
+
+
+def test(data, bot):
+    return 'issue' in data['message']
+
+
+def handler(data, bot, org_name='python-cn'):
+    repos = requests.get(REPO_API.format(org=org_name)).json()
+    rv = ''
+    for repo in repos:
+        repo_name = repo['name']
+        issues_count = repo['open_issues_count']
+        if issues_count != 0:
+            issues = requests.get(ISSUE_API.format(org=org_name,
+                                                   repo=repo_name)).json()
+            for issue in issues:
+                rv += issue['url'].encode('utf-8') + '\n'
+                rv += issue['title'].encode('utf-8') + '\n'
+                rv += issue['body'].encode('utf-8') + '\n'
+                rv += '\n'
+
+    return rv if rv != '' else 'no issue'
```

Comment:
about `handler` function:
* `if rv != ''` -> `if rv` or         
* `return rv or 'no issue'`

about plugin:
* add `github_` prefix for plugin/file name ?